web/package.json

```diff
@@ -8,51 +8,54 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@apidevtools/json-schema-ref-parser": "^9.0.1",
-    "@girder/components": "^2.2.5",
-    "@girder/oauth-client": "^0.7.6",
-    "@koumoul/vjsf": "1.21.0",
-    "@sentry/browser": "^5.20.1",
-    "@sentry/integrations": "^5.20.1",
-    "@vue/composition-api": "^1.0.0-beta.20",
-    "ajv": "^6.12.3",
-    "axios": "^0.19.2",
-    "core-js": "^3.6.5",
-    "filesize": "^6.1.0",
-    "js-yaml": "^3.14.0",
-    "lodash": "^4.17.19",
-    "moment": "^2.27.0",
+    "@apidevtools/json-schema-ref-parser": "^9.0.7",
+    "@girder/components": "~2.2.5",
+    "@girder/oauth-client": "^0.7.7",
+    "@koumoul/vjsf": "1.25.0-beta.12",
+    "@sentry/browser": "^6.3.6",
+    "@sentry/integrations": "^6.3.6",
+    "@vue/composition-api": "^1.0.0-rc.8",
+    "ajv": "^8.3.0",
+    "axios": "^0.21.1",
+    "core-js": "^3.12.1",
+    "filesize": "^6.3.0",
+    "js-yaml": "^4.1.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.1",
     "vue": "^2.6.11",
-    "vue-json-pretty": "^1.6.5",
-    "vue-router": "^3.3.4",
-    "vuetify": "^2.3.6",
-    "vuex": "^3.5.1",
+    "vue-json-pretty": "^1.8.0",
+    "vue-router": "^3.5.1",
+    "vuetify": "~2.3.6",
+    "vuex": "^3.6.2",
     "vuex-router-sync": "^5.0.0"
   },
+  "resolutions": {
+    "@girder/components/vuetify": "~2.3.6"
```

Comment:
`@girder/components` should probably be specifying `vuetify` as a `peerdependencies`. Since it doesn't, this pin will ensure that the top-level `vuetify` and its use nested within `@girder/components` are the same version.